test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -276,12 +276,17 @@ protected virtual void InitializeEnvironment(IHostEnvironment environment)
             environment.AddStandardComponents();
         }
 
-        /// <summary>
-        /// Runs a command with some arguments. Note that the input
-        /// <paramref name="toCompare"/> objects are used for comparison only.
-        /// </summary>
-        /// <returns>Whether this test succeeded.</returns>
         protected bool TestCore(RunContextBase ctx, string cmdName, string args, params PathArgument[] toCompare)
+        {
+            return TestCore(ctx, cmdName, args, DigitsOfPrecision, toCompare);
+        }
+
+            /// <summary>
+            /// Runs a command with some arguments. Note that the input
+            /// <paramref name="toCompare"/> objects are used for comparison only.
+            /// </summary>
+            /// <returns>Whether this test succeeded.</returns>
+         protected bool TestCore(RunContextBase ctx, string cmdName, string args, int digitsOfPrecision, params PathArgument[] toCompare)
```

Comment:
(nit) a single space was added to the beginning of the method, messing up the alignment.
Can you also align the `///` comments as well?

Reply:
The `///` comments are still not aligned.

---
In reply to: [224100533](https://github.com/dotnet/machinelearning/pull/1206#discussion_r224100533) [](ancestors = 224100533)